enterprise/models.py

```diff
@@ -4287,3 +4287,24 @@ class Meta:
         # ie no issue if multiple fields have: group = A and pending_enterprise_customer_user = NULL
         unique_together = (("group", "enterprise_customer_user"), ("group", "pending_enterprise_customer_user"))
         ordering = ['-modified']
+
+    def clean(self, *args, **kwargs):
+        """
+        Ensure that records added via Django Admin have matching customer records between learner and group.
+        """
+        user = self.enterprise_customer_user or self.pending_enterprise_customer_user
```

Comment:
nit: Duplicate line [here](https://github.com/openedx/edx-enterprise/pull/2044/files#diff-d950991245f878f58adc157895d6431af4385d522ab994aa25726e9abc2b434dR4309), maybe make into a helper property?

Reply:
yeah I think we can pull this out, good catch